src/__tests__/act.js

```diff
@@ -1,7 +1,5 @@
 import React from 'react'
-import {render, cleanup, fireEvent} from '../'
-
-afterEach(cleanup)
```

Comment:
Several of the tests were manually wiring up `cleanup` even though they didn't need to before (and they certainly don't need to now), so I fixed those.

Reply:
There a bunch of docs with `afterEach(cleanup)` too

Reply:
I made these unrelated polish changes to master so this PR could be slimmer.